nginx-php5/php-fpm.rc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -eu
+
+nginx
```

Comment:
You should create two separate service files: one for nginx (with `daemon off;`), one for fpm. Never fork off things in the background.